README.md

```diff
@@ -7,30 +7,73 @@ Uses [dolthub/swiss](https://github.com/dolthub/swiss) map implementation under
 ## Installation
 
 Supports 1.18+ Go versions because of Go Generics
+
 ```
 go get github.com/mhmtszr/concurrent-swiss-map
 ```
 
 ## Usage
+
 New functions will be added soon...
+
 ```go
+package main
+
+import (
+	"hash/fnv"
 
-myMap := csmap.Create[int, string](
-    csmap.WithShardCount[int, string](32), // default 32,
-    csmap.WithCustomHasher[int, string](func(key int) uint64 {
-        return 0
-    }), // default maphash,
-    csmap.WithSize[int, string](1000), // default 0
+	csmap "github.com/mhmtszr/concurrent-swiss-map"
 )
-myMap.Store(10, "test")
-myMap.Load(10)
-myMap.Delete(10)
-myMap.Has(10)
-myMap.IsEmpty()
-myMap.SetIfAbsent(10, "test")
-myMap.Range(func(key int, value string) (stop bool) {})
-myMap.Count()
 
+func main() {
+	myMap := csmap.Create[string, int](
+		// set the number of map shards. the default value is 32.
+		csmap.WithShardCount[string, int](32),
+
+		// if don't set custom hasher, use the built-in maphash.
+		csmap.WithCustomHasher[string, int](func(key string) uint64 {
+			hash := fnv.New64a()
+			hash.Write([]byte(key))
+			return hash.Sum64()
+		}),
+
+		// set the default capacity for each map. the default value is 0.
```

Comment:
Actually, this size is the total capacity of shard maps. We divide this value into shard counts to find the capacity for each map.

Reply:
done, please cr again.

Reply:
@rfyiamcool I think it is still misleading what about 

// set the total capacity, every shard map has total capacity/shard count capacity. the default value is 0.

Reply:
You are right, my comment was expressed incorrectly. 😁